external/avro/src/main/scala/org/apache/spark/sql/avro/AvroFileFormat.scala

```diff
@@ -276,10 +274,15 @@ private[avro] object AvroFileFormat {
     }
   }
 
-  def ignoreFilesWithoutExtensions(conf: Configuration): Boolean = {
-    // Files without .avro extensions are not ignored by default
-    val defaultValue = false
+  def ignoreExtension(conf: Configuration, options: Map[String, String]): Boolean = {
```

Comment:
Could we have a class `AvroOptions` like what we are doing for the other built-in data sources?

Reply:
Would you like to see it as a part of this PR or a separate one? I would extract some common code like `getBool()` from CSVOptions to a separate trait and extend `AvroOptions` by it.